bundles/framework/layerselection2/resources/css/style.css

```diff
@@ -55,11 +55,22 @@ div.layerselection2 .layer.selected {
           padding-top: 2px; }
         div.layerselection2 .layer.selected div.layer-tools div.right-tools div.object-data a {
           padding-left: 5px; }
+        div.layerselection2 .layer.selected div.layer-tools div.right-tools div.object-data div.show-metadata-tool {
```

Comment:
When metadata tool is added by metadataflyout bundle I'd say the icon/css for the icon should be on that bundle instead of layerselection.